functions/submit_user.js

```diff
@@ -9,9 +11,31 @@ module.exports.handler = (event, context, callback) => {
     event: JSON.stringify(event),
   });
 
-  mailchimp.post(`/lists/${process.env.MAILCHIMP_LIST_ID}/members`, {
-    email_address: event.body.email,
-    status: 'subscribed'
+  Promise.promisify(civicinfo.representatives.representativeInfoByAddress)({
```

Comment:
`promisify` isn't a native part of the Promise spec, however you can wrap things that have callbacks pretty eaily:

```
new Promise((resolve, reject) => {
  // function myCallbackThing(param, success, error) { ... }
  myCallbackThing(param, resolve, reject)
})
.then(data => ...)
.catch(error => ...)
```